mochify/index.js

```diff
@@ -18,26 +18,40 @@ async function mochify(options = {}) {
   const mocha_runner = createMochaRunner(config.reporter || 'spec');
   const { mochifyDriver } = resolveMochifyDriver(config.driver);
 
+  const [mocha, client, files] = await Promise.all([
+    readFile(require.resolve('mocha/mocha.js'), 'utf8'),
+    readFile(require.resolve('./client'), 'utf8'),
+    resolveSpec(config.spec)
+  ]);
+
+  const configured_client = setupClient(client, config);
```

Comment:
It seems we previously never passed `config` to that call so I added it as a drive by.